conf/janus.plugin.videoroom.cfg.sample

```diff
@@ -18,5 +18,6 @@ publishers = 6
 bitrate = 128000
 fir_freq = 10
 ;videocodec = vp8
+;audiocodec = opus
```

Comment:
Can you put the audiocodec stuff before the videocodec stuff? Throughout all the code I mean (config as in here, when parsing, when printing, etc.). This is the convention we follow in all the gateway.